balancer/grpclb/grpc_lb_v1/load_balancer_grpc.pb.go

```diff
@@ -61,18 +61,22 @@ func (x *loadBalancerBalanceLoadClient) Recv() (*LoadBalanceResponse, error) {
 }
 
 // LoadBalancerServer is the server API for LoadBalancer service.
+// All implementations must embed UnimplementedLoadBalancerServer
+// for forward compatibility
 type LoadBalancerServer interface {
 	// Bidirectional rpc to get a list of servers.
 	BalanceLoad(LoadBalancer_BalanceLoadServer) error
+	mustEmbedUnimplementedLoadBalancerServer()
```

Comment:
Oh, we said we want to revert the changes to the generated code, because they are API changes.